frontend/packages/dev-console/src/components/import/deployImage-submit-utils.ts

```diff
@@ -278,18 +281,17 @@ export const createResources = async (
   } else if (!dryRun) {
     // Do not run serverless call during the dry run.
     const [imageStreamResponse] = await Promise.all(requests);
-    requests.push(
-      createKnativeService(
-        name,
-        applicationName,
-        projectName,
-        scaling,
-        limits,
-        unknownTargetPort,
-        userLabels,
-        imageStreamResponse.status.dockerImageRepository,
-      ),
+    const knDeploymentResource = getKnativeServiceDepResource(
```

Comment:
Why not send the whole `formData` instead of so many params and let the function decide what it wants to use? This is what we're doing currently for all resource creation utils in import and deploy forms.